components/chat-panel.tsx

```diff
@@ -63,7 +63,7 @@ export function ChatPanel({
               <div
                 key={example.heading}
                 className={cn(
-                  'cursor-pointer bg-zinc-50 text-zinc-950 rounded-2xl p-4 sm:p-6 hover:bg-zinc-100 transition-colors',
+                  'cursor-pointer bg-secondary rounded-2xl p-4 sm:p-6 transition-colors',
```

Comment:
```suggestion
                  'cursor-pointer bg-background rounded-2xl p-4 sm:p-6 transition-colors',
```

you are removing the colour distinction of the button if not using something more distinct such as bg-background